docs/cloud/pouta/multiattach.md

```diff
@@ -0,0 +1,499 @@
+
+    ```fstab
+    LABEL=gfs_cluster:mygfs2 /mnt gfs2 defaults 0 0
+    ```
+
+### GFS2 FAQ
+
+* **How to add more nodes?**
+
+    It is possible to add new nodes to a GFS2 cluster. The supported **limit** is **16** nodes.
+
+    First you need to make sure there are enough journal entries. Use `gfs2_edit` to get the total number of journals:
+
+    ```sh
+    sudo gfs2_edit -p jindex /dev/vdb | grep journal
+    ```
+
+    If it is not enough, you can easily add more with `gfs2_jadd`: 
+
+    ```sh
+    $ sudo gfs2_jadd -j 1 /mnt
+    Filesystem: /mnt
+    Old journals: 15
+    New journals: 16
+    ```
+
+    Secondly, create the new node, install the required software and attach the volume using openstack API. The process is described above.
+
+    Then you need to edit the file `/etc/corosync/corosync.conf` in every node and add an entry for the new one:
+
+    ```json
+    node {
+     ring0_addr: cinder-gfs2-16
+     nodeid: 16
+    }
+    ```
+
+    Once the file is updated, you need to stop the mount and restart the dlm and corosync daemons in every node in the cluster.
+
+    Finally, you just need to mount the volume:
+
+    ```sh
+    sudo mount -L gfs_cluster:mygfs2 /mnt
+    ```
+
+    It is recommended to add an entry to `/etc/fstab` so the volume is mounted automatcally when rebooted. For example adding a line like this:
+
+    ```fstab
+    LABEL=gfs_cluster:mygfs2 /mnt/ gfs2 defaults 0 0
+    ```
+
+* **What happens if a VM gets disconnected?**
+
+    This covers two different use cases, a temporal and/or unexpected disconnection, and a permanent one.
+
+    For a temporal and unexpected disconnection, the cluster should be able to deal with this kind of issues automatically. After the node is back, you need to check that all came back to normal. In some cases the automatic mount of the volume can fail, if so mount the volume as explained above.
+
+    If it is temporal but expected, like to update the kernel version. Umount the volume in the node (`sudo umount /mnt`) before rebooting the node. It is not required, but recommended.
+
+    For a permanent disconnection of a VM, one need to do the inverse process of adding a new node. Umount the volume (`sudo umount /mnt`), remove the entry for this VM in the `/etc/corosyncecorosync.conf` file of every node, and finally restart the daemons in every node. This needs to be done as it affects the quorum count for the cluster.
+
+* **Is it possible to mount a node as read-only?**
+
+    Yes, GFS2 has the "spectator mode":
+
+    ```
+    spectator
+       Mount  this filesystem using a special form of read-only mount.  The mount does not
+       use one of the filesystem's journals. The node is unable to  recover  journals  for
+       other nodes.
+
+    norecovery
+       A synonym for spectator
+    ```
+
+    So, just run this command:
+
+    ```sh
+    sudo mount /dev/vdb /mnt -o spectator
```

Comment:
Should we precise the type?
```
sudo mount /dev/vdb /mnt -t gfs2 -o spectator
```

Reply:
Thinking....

Reply:
Ok, let's add it, with a note that it is not necessary, but recommended to make sure the correct device is mounted.

What do you think?

Reply:
Agree 👍